make_jetson_nano.sh

```diff
@@ -2,6 +2,9 @@
 
 version=R32.5.0
 release=5.0
+script_dir=`pwd`
```

Comment:
pwd doesn't give you what you want, it would just be your current directory not the script directory. Can do $(dirname $0) or 
pushd ${BASH_SOURCE%/*}/
script_dir=`pwd`
popd

Reply:
Check it out now. Switched to using `dirname` and apparently back-ticks are out-of-date, so `$()` is the way to go.